lua/wikis/commons/MatchGroup/Input/Util.lua

```diff
@@ -834,10 +834,6 @@ function MatchGroupInputUtil.matchIsFinished(match, maps, opponents)
 		return true
 	end
 
-	if not MatchGroupInputUtil.hasScore(opponents) then
```

Comment:
Not sure if we want to remove it, or instead move it to get checked after the time?